llama-index-integrations/vector_stores/llama-index-vector-stores-weaviate/pyproject.toml

```diff
@@ -32,7 +32,7 @@ version = "0.1.5"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.10.1"
-weaviate-client = "^3.26.2"
+weaviate-client = "^4.5.7"
```

Comment:
We should bump the package version above to like v0.2.0 ?

Reply:
Done! I thought we'd do a version bump in a separate PR as part of the release process.

Reply:
This just crossed my mind:

Why v0.2.0 instead of v1.0.0? From a weaviate client's perspective, we don't have plans to introduce breaking changes like we did from v3 to v4 so I would consider this a stable release, unless the LlamaIndex team is considering rearchitecting the vector store interface in the near future.

Reply:
I think all the integrations started out at v0.1.0

But, its weaviates integration, so you are free to version it how you please :)

Reply:
then let's go with v1.0.0! 🚀